ports/mimxrt/boards/make-flexram-config.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+"""Evaluate FlexRAM configuration and generate startup code."""
```

Comment:
what is the license/copyright of this file?

Reply:
It was written by @alphaFred, so I assume it's MIT, and Philipp will add the appropriate License.

Reply:
Yes - I will add the license

Reply:
please make the style of the license/copyright match existing files, eg `tools/pyboard.py` and `tools/mpy_ld.py` (use `#` at the start of the line)